[julia-python-homework]/3.conditional_logic/1.secure_password_generator.py

```diff
@@ -0,0 +1,15 @@
+"""Create an app which checks user's password
+and based on your rules states if it's secure"""
+
+import re
+
+password = input("Enter your password: ")
+
+if len(password) >= 8 and re.search(r"\d", password):
+    print("Strong password")
+
+elif len(password) >= 8:
+    print("Medium password")
+
+else:
+    print("Weak password")
```

Comment:
We haven't learnt `regex` yet during this lesson :)

Implementation is fine, and very nice that you've picked it up. But during this stage, I was expecting something like this:

```python

password_strenght = 0

for letter in password:
    if letter in ['$', '#' , '@']: 
       password_strenght +=1         

if password_strenght >=2: 
   ...
```